public/install.php

```diff
@@ -66,6 +66,14 @@
 			<?php endif ?>
 		</tr>
 		<tr>
+			<th>Composer Installed</th>
+			<?php if (is_dir($vendor_path) AND is_file($vendor_path.'autoload.php')): ?>
+				<td class="pass"><?php echo $vendor_path ?></td>
+			<?php else: $failed = TRUE ?>
+				<td class="fail">Kohana requires Composer to be installed, <a href="http://getcomposer.org">get it</a> and call <code>composer install</code>.</td>
```

Comment:
This anchor should really use `target="_blank"` so the user doesn't leave the installation page.